File: app/src/main/java/com/kkkkkn/readbooks/view/viewHolder/BookChaptersAdapter_ViewHolder.java
package com.kkkkkn.readbooks.view.viewHolder;

import android.view.View;
import android.widget.TextView;

import androidx.appcompat.widget.AppCompatTextView;

import com.kkkkkn.readbooks.R;

public class BookChaptersAdapter_ViewHolder {
    public AppCompatTextView chapterName;
    public BookChaptersAdapter_ViewHolder(View viewRoot) {
        chapterName=(AppCompatTextView) viewRoot.findViewById(R.id.book_info_chapter_item_chapterName);
    }
}

File: app/src/main/java/com/kkkkkn/readbooks/view/viewHolder/BookChaptersAdapterViewHolder.kt
package com.kkkkkn.readbooks.view.viewHolder

import android.view.View
import androidx.appcompat.widget.AppCompatTextView
import androidx.recyclerview.widget.RecyclerView
import com.kkkkkn.readbooks.R

class BookChaptersAdapterViewHolder(viewRoot: View, type: Int) :
    RecyclerView.ViewHolder(viewRoot) {
    var chapterName: AppCompatTextView

    init {
        chapterName =
            viewRoot.findViewById<View>(R.id.book_info_chapter_item_chapterName) as AppCompatTextView
    }
}